Vienna/Sources/Fetching/OpenReader.h

```diff
@@ -20,17 +20,19 @@
 
 // Check if an accessToken is available
 @property (nonatomic, getter=isReady, readonly) BOOL ready;
+@property (nonatomic) NSOperation * unreadCountOperation;
 
 -(void)loadSubscriptions;
 -(void)clearAuthentication;
 -(void)resetAuthentication;
 
--(void)subscribeToFeed:(NSString *)feedURL;
--(void)unsubscribeFromFeed:(NSString *)feedURL;
+-(void)subscribeToFeed:(NSString *)feedURL withLabel:(NSString *)label;
+-(void)unsubscribeFromFeed:(NSString *)feedIdentifier;
 -(void)markRead:(Article *)article readFlag:(BOOL)flag;
 -(void)markStarred:(Article *)article starredFlag:(BOOL)flag;
--(void)setFolderLabel:(NSString *)folderName forFeed:(NSString *)feedURL set:(BOOL)flag;
--(void)setFolderTitle:(NSString *)folderName forFeed:(NSString *)feedURL;
+-(void)markAllRead:(Folder *)folder;
```

Comment:
How about `markAllReadInFolder`? or even creating a method on the Folder object to mark all read?